app/src/main/java/com/dailyaims/screens/splash/home/HomeFragment.kt

```diff
@@ -0,0 +1,45 @@
+package com.dailyaims.screens.splash.home
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.dailyaims.Aims
+import com.dailyaims.databinding.FragmentHomeBinding
+import com.dailyaims.screens.splash.recycler_view_adapter.AimsAdapter
+
+class HomeFragment : Fragment() {
```

Comment:
when you create a fragment, create viewModel too, and try to replace 'logic' from fragment to viewModel

Reply:
in our case, get "Aims" list from ViewModel and when click on FAB tell viewModel to add new "Aim" to list